src/util.js

```diff
@@ -27,3 +27,44 @@ export function isString(value) {
 export function isFunction(value) {
   return typeof value === 'function';
 }
+
+/**
+ *  This Function Creates element based on type passed
+ *  currently limited to Link and script element
+ *
+ * @param {*} url
+ * @param {*} type
+ * @returns
+ */
+export function createElement(url, type) {
+  let element = null;
+
+  if (type === 'text/css') {
+    element = document.createElement('link');
+    element.rel = 'stylesheet';
+    element.type = 'text/css';
+    element.href = url;
+  } else if (type === 'text/javascript') {
+    element = document.createElement('script');
+    element.type = 'text/javascript';
```

Comment:
Instead of hard-coding `text/javascript` in multiple places - can you define constants at the top and use it everywhere? 

Like this:
```js
const TYPE_CSS = 'text/css';
const TYPE_JAVASCRIPT = 'text/javascript';

...

if (type === TYPE_JAVASCRIPT) {
  ...
```